site/soc/software/charts/osh/openstack-tempest/tempest.yaml

```diff
@@ -0,0 +1,100 @@
+{% if deploy_tempest is defined and deploy_tempest is sameas true %}
+  values:
+    # Adding images overrides here instead of versions.yaml
+    images:
+      tags:
+        tempest_run_tests: "{{ suse_osh_registry_location }}/openstackhelm/tempest:{{ suse_infra_image_version }}"
+        ks_user: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_openstack_image_version }}"
+    conf:
+      script: "tempest run --config-file /etc/tempest/tempest.conf -w 4 {{ test_args }}"
+      tempest:
+        compute:
+          flavor_ref: "{{ tempest_test_flavor_id.stdout }}"
+          image_ref: "{{ tempest_test_image_id.stdout }}"
+          image_ref_alt: "{{ tempest_test_image_id.stdout }}"
+          hypervisor_type: "{{ tempest_test_hypervisor_type | default('qemu') }}"
+        network:
+          default_network: "10.0.0.0/8"
+          project_network_cidr: "{{ openstack_project_network_cidr }}"
+          floating_network_name: "{{ openstack_external_network_name }}"
+          public_network_id: "{{ tempest_test_public_network_id.stdout }}"
+        validation:
+          image_ssh_user: "cirros"
+          image_ssh_password: "gocubsgo"
+          network_for_ssh: "{{ openstack_external_network_name }}"
+          floating_ip_range: "{{ openstack_external_subnet_cidr.stdout }}"
+        service_available:
+          cinder: "{{ tempest_enable_cinder_service }}"
+          glance: "{{ tempest_enable_glance_service }}"
+          nova: "{{ tempest_enable_nova_service }}"
+          neutron: "{{ tempest_enable_neutron_service }}"
+    endpoints:
+      cluster_domain_suffix: cluster.local
+      local_image_registry:
+        name: docker-registry
+        namespace: docker-registry
+        hosts:
+          default: localhost
+          internal: docker-registry
+          node: localhost
+        host_fqdn_override:
+          default: null
+        port:
+          registry:
+            node: 5000
+      identity:
+        name: keystone
+        auth:
+          admin:
+            region_name: RegionOne
+            username: admin
+            password: {{ lookup('password', secrets_location + '/osh_keystone_admin_password ' + password_opts) }}
+            project_name: admin
+            user_domain_name: default
+            project_domain_name: default
+          tempest:
+            role: admin
+            region_name: RegionOne
+            username: tempest
+            password: {{ lookup('password', secrets_location + '/osh_tempest_password ' + password_opts) }}
+            project_name: service
+            user_domain_name: service
+            project_domain_name: service
+        hosts:
+          default: identity
+          internal: identity-api
+        host_fqdn_override:
+          default: null
+        path:
+          default: /v3
+        scheme:
+          default: http
+        port:
+          api:
+            default: 80
+            internal: 5000
+{% endif %}  
+  source:
+    type: git
```

Comment:
chart source and version should also be defined in the versions.yaml and use substitution to override here.

should not use master version, pin it down to a commit sha.

Reply:
I pinned the reference to a commit sha, and moved chart and image info to versions.yml. I think I originally put this here while testing and forgot to move it out again.